lib/bip_claims/service.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module BipClaims
+  class Service < Common::Client::Base
+    STATSD_KEY_PREFIX = 'api.bip_claims'
+    include Common::Client::Monitoring
+
+    configuration BipClaims::Configuration
+
+    def veteran_attributes(claim)
+      case claim.form_id
+      when '21P-530'
+        ssn, full_name, bday = claim.parsed_form.values_at(
+          'veteranSocialSecurityNumber',
+          'veteranFullName',
+          'veteranDateOfBirth'
+        )
+      end
+
+      BipClaims::Veteran.new(
```

Comment:
Would you want a BipClaims::Veteran with a nil ssn or bday (if claim.form_id != '21P-530' )?

Reply:
I was anticipating having to support some other type of forms in the future but the timing on those are undetermined.

Would you recommend raising an error for "unsupported form id" or just returning a nil veteran? For this service, we expect only `21P-530` forms for the indefinite future

Reply:
@annaswims

I was anticipating having to support some other type of forms in the future but the timing on those are undetermined.

Would you recommend raising an error for "unsupported form id" or just returning a nil veteran? For this service, we expect only `21P-530` forms for the indefinite future